src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -717,6 +717,283 @@ For example, if the page that was copied from is called `Leads`, then the newly
 }
 ----
 
+[#setting-page-icon]
+== Setting page Icon
+
+Icon can be set to a record list or record page.
+This gives us the ability to further customize a page and easily distinguish it from other pages.
+
+To set a page icon, navigate to its `Edit page` view.
+It can be accessed in the admin area of Corteza under pages in the sidebar.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-public-view.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "x": 15,
+    "y": 135,
+    "w": 290,
+    "h": 40
+  }]
+}
+----
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-tree-case-management.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "x": 1520,
+    "y": 225,
+    "w": 70,
+    "h": 25
+  }]
+}
+----
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit.png",
+    "alias": "page-edit-base",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "x": 590,
+    "y": 440,
+    "w": 95,
+    "h": 60
+  }]
+}
+----
+
+Click on the icon to set page icon.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit.png",
+    "alias": "base-set-icon",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 555,
+    "y": 150,
+    "w": 1105,
+    "h": 685
+  },
+  "annotations": [{
+    "x": 670,
+    "y": 450,
+    "w": 10,
+    "h": 13
+  }]
+}
+----
+
+A modal will open with two sub-sections: `Upload icon` and `Or add link to icon`.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-icon.png",
+    "alias": "page-edit-configure-icon-view-modal",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 570,
+    "y": 40,
+    "w": 780,
+    "h": 330
+  }
+}
+----
+
+[NOTE]
+====
+The currently supported icon format is `.icon`.
+`.svg` isn't supported yet.
+====
+
+In the `Upload icon` sub-section you can drag and drop or click on the button and choose an icon.
+In the `Or add link to icon` sub-section you can add a link to an icon and preview it.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-icon.png",
+    "alias": "page-edit-configure-icon-preview",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 570,
+    "y": 40,
+    "w": 780,
+    "h": 330
+  },
+  "annotations": [{
+    "x": 1300,
+    "y": 240,
+    "w": 40,
+    "h": 40
+  }]
```

Comment:
Consider annotating the entire input and not just the button like so:

![image](https://user-images.githubusercontent.com/12545711/232429071-2856b4e0-3e43-4530-baf9-14c30b7831ac.png)